docs/public/upgrading/gui.md

```diff
@@ -0,0 +1,27 @@
+# Upgrade GUI
+
+## Overview
+
+This document documents manual steps needed to keep customizations, if any, from
+one version of Zonemaster-GUI to a later version. If no such customizations exist
+nothing special has to be done.
+
+| Current Zonemaster-GUI version | Comments                                                 |
+|--------------------------------|----------------------------------------------------------|
+| version < v5.0.0               | Any `assets/app.config.json` could be migrated to v5.0.0 |
```

Comment:
I don't understand how this table is supposed to be used. To me it looks like a different version of what comes below, but with different formatting, fewer details, and no link to where the details can be found. To me it would make sense to replace it with a list of links to the migrations, like a kind of table of contents.

```suggestion
Migrations:
 * [pre-v5.0.0 to v5.0.0](#migrating-pre-v5.0.0-to-v5.0.0)
```